lib/web/desktop.go

```diff
@@ -353,8 +371,17 @@ func proxyWebsocketConn(ws *websocket.Conn, wds net.Conn) error {
 			if utils.IsOKNetworkError(err) {
 				errs <- nil
 				return
-			}
-			if err != nil {
+			} else if tdp.IsNonFatalErr(err) {
```

Comment:
Can we simplify these 2 branches that are almost identical?

```
isFatal := !tdp.IsNonFatalErr(err)
sendTDPNotification(ws, err, isFatal)
```

Reply:
https://github.com/gravitational/teleport/blob/700fff47d6f25fbc974b6ce6618b899b28cbbd4f/lib/web/desktop.go#L372-L385